eclipse-theia/theia
# upgrade ws to 8.17.1 & jsdom to 24.1.0

#### What it does
Fixes #13848

This is a simple update on dependent library, known to have a risk of DoS. I tested the tool, no changes in behavior, and I also di not notice any changes in the logs.



#### How to test
Since this updates websocket lib, build and open the tool. Observe the logs to make sure we don't get any new entries.

#### Follow-ups



#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
